### DISCUSSION TITLE
Translation of Commercial modules?

### DISCUSSION THREAD

**Discussion by improwise** [abpframework/abp#20424] [Q&A]
Just created a Push Request for translating the ABP framework to Swedish. That should hopefully work for the free stuff, but what about things that only exists in the Commercial variant which I assume won't be translated by this approach, how would you do the same there?

**Comment by realLiangshiwei** [comment]
Hi,

We decided to keep only English documents to make sure they are always up to date.

Others who want to read the translation can use the Translate button.

<img width="1365" alt="image" src="https://github.com/user-attachments/assets/c8d5cfbb-563c-4239-b807-0654956f3790">

**Reply by improwise** [reply]
Hi,

I am mainly thinking of the UI, error messages etc of those commercial components rather than the documentation.
